### PR TITLE
added goldsam's features repo to the index

### PR DESCRIPTION
**Pull request by goldsam**
Adding my own features to the index

### CODE REVIEW

**Review by eljog — COMMENTED** [review]
Thank you the PR

For visibility, could you please add some info about the features that add being added?

**Comment by goldsam** [comment]
I updated the README.md in [my repo](https://github.com/goldsam/dev-container-features) to summarize each feature

File: _data/collection-index.yml [hunk 1]
```diff
@@ -648,3 +648,8 @@
   contact: https://github.com/CASL0/devcontainer-features/issues
   repository: https://github.com/CASL0/devcontainer-features
   ociReference: ghcr.io/casl0/devcontainer-features
+- name: Some Additional Development Container Features
```
Comment (eljog) [inline _data/collection-index.yml]: Please consider using a more specific/unique name here, in line with the rest of the entries in this file

**Review by eljog — COMMENTED** [review]
Added a comment. Looks good otherwise

**Comment by goldsam** [comment]
I updated the name to be similar to that of the entry directly above mine

**Review by eljog — APPROVED** [review]
⚡

**Pull request merged**